app/models/search/tag_search_form.rb

```diff
@@ -30,4 +35,29 @@ def search_results
     @searcher.search_results
   end
 
+  def set_fandoms
+    return if @options[:fandoms].blank?
+
+    names = @options[:fandoms].split(",").map(&:squish)
+    @options[:fandom_ids] = Tag.where(name: names).pluck(:id)
+  end
+
+  def sort_columns
+    options[:sort_column] || "name"
+  end
+
+  def sort_direction
+    options[:sort_direction] || default_sort_direction
+  end
+
+  def sort_options
+    [
+      %w[Name name],
+      ["Date Created", "created_at"]
+    ]
+  end
+
+  def default_sort_direction
+    %w[name].include?(sort_column) ? "asc" : "desc"
```

Comment:
I'm not sure if this is working as intended. When I go to the tag search page, it's set to Name and Descending by default, so I get results Z-A instead of A-Z.